src/nvim/statusline.c

```diff
@@ -90,89 +103,11 @@ void win_redr_status(win_T *wp)
     wp->w_redr_status = true;
   } else if (*p_stl != NUL || *wp->w_p_stl != NUL) {
     // redraw custom status line
-    redraw_custom_statusline(wp);
+    redraw_custom_statusline(wp, NULL);
```

Comment:
Instead of passing `NULL` when the default format is not being used and the default format otherwise. Why not just always pass the statusline string that will be used (user-configured statusline string if it exists and default statusline string otherwise) instead of letting `redraw_custom_statusline` choose the statusline string when `NULL` is passed?

Reply:
I wanted to still include a fallback if the fmt string was NULL, so that's why I did this. But makes sense! I'll change that when I start my work on this PR again.